scripts/containers/BucketCollectionsPage.js

```diff
@@ -0,0 +1,28 @@
+import { bindActionCreators } from "redux";
+import { connect } from "react-redux";
+import BucketCollections from "../components/BucketCollections";
```

Comment:
Nit: We usually separate dependency imports from project ones with an empty line.